src/prommis/uky/uky_flowsheet.py

```diff
@@ -861,5 +873,839 @@ def display_results(m):
+        costing_method_arguments={
+            "cost_accounts": CSX_tank_mixer_accounts,
+            "scaled_param": m.fs.CSX_tank_mixers.power,
+            "source": 1,
+            "n_equip": 2,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # 6.3 is UKy Cleaner Solvent Extraction - Process Pump
+    CSX_pump_accounts = ["6.3"]
+    m.fs.CSX_pump = UnitModelBlock()
+    flow_6_3 = value(
+        units.convert(
+            flowsheet.fs.solex_cleaner.mscontactor.aqueous_inlet.flow_vol[0],
+            to_units=units.gal / units.min,
+        )
+    )
+    m.fs.CSX_pump.feed_rate = Var(initialize=flow_6_3, units=units.gal / units.min)
+    m.fs.CSX_pump.feed_rate.fix()
+    m.fs.CSX_pump.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": CSX_pump_accounts,
+            "scaled_param": m.fs.CSX_pump.feed_rate,
+            "source": 1,
+            "n_equip": 3,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # 6.4 is UKy Cleaner Solvent Extraction - Mixer Settler
+    CSX_mixer_settler_accounts = ["6.4"]
+    m.fs.CSX_mixer_settler = UnitModelBlock()
+    m.fs.CSX_mixer_settler.volume = Var(initialize=24.44, units=units.gal)
+    m.fs.CSX_mixer_settler.volume.fix()
+    m.fs.CSX_mixer_settler.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": CSX_mixer_settler_accounts,
+            "scaled_param": m.fs.CSX_mixer_settler.volume,
+            "source": 1,
+            "n_equip": 6,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # Precipitation costs
+    # 9.2 is UKy Rare Earth Element Precipitation - Polyethylene Tanks
+    reep_pe_tanks_accounts = ["9.2"]
+    m.fs.reep_pe_tanks = UnitModelBlock()
+    m.fs.reep_pe_tanks.capacity = Var(initialize=15.04, units=units.gal)
+    m.fs.reep_pe_tanks.capacity.fix()
+    m.fs.reep_pe_tanks.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=QGESSCostingData.get_REE_costing,
+        costing_method_arguments={
+            "cost_accounts": reep_pe_tanks_accounts,
+            "scaled_param": m.fs.reep_pe_tanks.capacity,
+            "source": 1,
+            "n_equip": 1,
+            "scale_down_parallel_equip": False,
+            "CE_index_year": CE_index_year,
+        },
+    )
+
+    # 9.3 is UKy Rare Earth Element Precipitation - Tank Mixer
+    reep_tank_mixer_accounts = ["9.3"]
+    m.fs.reep_tank_mixers = UnitModelBlock()
+    m.fs.reep_tank_mixers.power = Var(initialize=0.61, units=units.hp)
```

Comment:
This is another one of the situations where I felt like it didn't make sense to reduce the power to 0.0061. However, I don't have a very strong intuitive sense of what the power of a tank mixer should be, so feel free to let me know what a reasonable value is.

Reply:
@Agfritz @bpaul4 We really need input from the costing team on good values for these. @MarcusHolly has experience with unit model development, but not in costing so we need guidance on these sorts of things.

Reply:
Thanks @MarcusHolly and @andrewlee94 , I agree that 0.0061 is not reasonable for hp (I would assume it would be somewhere between 0.2-0.75) so I think this is reasonable for now.  This is another case where, in reality, the pilot team may just use whatever equipment they have available even if the sizing is not perfect. When we are scaling these up ideally the hp, tank size, and pump flow rate would all be linked and calculated in the unit model.